toptal/picasso
# feat(Icon): add new `flag` and `leadOrgUnitJoin`  icons from BASE

[TI-1216](https://toptal-core.atlassian.net/browse/TI-1216)

### Description

We need to add `Flag` and `LeadOrgUnitJoin` icons in scope of this task.

This icons are part of our BASE Misc icons ([link](https://www.figma.com/file/TqaGgbpjGSUDf7qq153Isq/Product-Iconography?node-id=1-784&t=JqABJMxA38NQYcCG-4))

<img width="935" alt="image" src="https://user-images.githubusercontent.com/1213542/231533075-f78d7bcf-01a1-4f23-90a1-3c64dc35f61d.png">

<img width="1639" alt="image" src="https://user-images.githubusercontent.com/1213542/231536699-d0b105e7-5ead-4a3d-b496-a49e195c21d9.png">

### How to test

- `yarn start`
- Go to Icons
- Search for `flag`
- `Flag16` and `Flag24` should be there
- Search for `lead`
- `LeadOrgUnitJoin16` and `LeadOrgUnit24` should be there

### Screenshots

<img width="1792" alt="image" src="https://user-images.githubusercontent.com/1213542/231533168-7aaff4f6-6294-4da1-a506-2b51410da08e.png">

<img width="1792" alt="image" src="https://user-images.githubusercontent.com/1213542/231537047-1ae1efe3-8779-4b9e-a73c-7804b1d8065f.png">


### Development checks

- ~Annotate all `props` in component with documentation~
- ~Create `examples` for component~
- ~Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).~
- ~Covered with tests~

> All **_development checks_** should be done and set checked to pass the
> **GitHub Bot: TODOLess** action

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run package:alpha-release` - Release alpha version
- `@toptal-anvil ping reviewers` - Ping FX team for review

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>


[TI-1216]: https://toptal-core.atlassian.net/browse/TI-1216?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
